docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -15,7 +15,7 @@ kyma alpha deploy [flags]
 ## Flags
 
 ```bash
-  -a, --atomic                    Use atomic deployment, which rolls back any component that could not be installed successfully. (default true)
+  -a, --atomic                    Use atomic deployment, which rolls back any component that could not be installed successfully.
```

Comment:
```suggestion
  -a, --atomic                    Use atomic deployment, which rolls back any component that could not be installed successfully (default: false).
```

Reply:
I can incorporate that to the description, but this is the same thing as happened with Mostafa's PR earlier.
This `(default XXX)` is generated by the tool.

Reply:
then it's fine! Didn't know that